### PR TITLE
List

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (444)
Screenshot (445)
Screenshot (446)
Screenshot (447)
Added List component with spaced list, stacked list with and without bullets and ordered list.

### CODE REVIEW

**Review by stutisk — COMMENTED** [review]
code looks good to me

File: assets/common-css/navbar.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+.navbar-container{
+    padding: 20px;
+    /* display: flex; */
```
Comment (omgaikwad) [inline assets/common-css/navbar.css:4]: you can remove the comments

File: assets/common-css/navbar.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+.navbar{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.lib-name{
+    text-decoration: underline var(--secondary-color);
+    /* font-size: 1.3rem; */
```
Comment (omgaikwad) [inline assets/common-css/navbar.css:13]: here also, remove comments

**Review by omgaikwad — COMMENTED** [review]
Everything else good to me.

**Pull request merged**